macos/Sources/Helpers/Fullscreen.swift

```diff
@@ -141,6 +145,7 @@ class NonNativeFullscreen: FullscreenBase, FullscreenStyle {
 
     struct Properties {
         var hideMenu: Bool = true
+        var untitled: Bool = true
```

Comment:
I think having an inverted bool here makes the cognitive load slightly higher versus matching the config. I'd change this instead to `titled` and invert all the logic so we match the config.